From 77749e43ced473de311d8407f352009cc5b073a2 Mon Sep 17 00:00:00 2001
From: Phil Sutter <phil@nwl.cc>
Date: Tue, 23 Jul 2019 15:24:41 +0200
Subject: [PATCH] doc: Install ip{6,}tables-restore-translate.8 man pages

Just like in b738ca3677785 ("doc: Install ip{6,}tables-translate.8
manpages"), create man pages for *-restore-translate tools as semantic
links to xtables-translate.8.

Signed-off-by: Phil Sutter <phil@nwl.cc>
Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
(cherry picked from commit 3dfb01cf14d72c21a15ab69a2767280223b7e3cb)

Conflicts:
	iptables/Makefile.am
-> Patch applied manually, multiple context changes.

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 iptables/.gitignore  | 2 ++
 iptables/Makefile.am | 3 ++-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/iptables/.gitignore b/iptables/.gitignore
index c638139b8a1d0..d46adc8a32f02 100644
--- a/iptables/.gitignore
+++ b/iptables/.gitignore
@@ -3,6 +3,7 @@
 /ip6tables-restore
 /ip6tables-static
 /ip6tables-translate.8
+/ip6tables-restore-translate.8
 /iptables
 /iptables.8
 /iptables-extensions.8
@@ -13,6 +14,7 @@
 /iptables-restore.8
 /iptables-static
 /iptables-translate.8
+/iptables-restore-translate.8
 /iptables-xml
 /iptables-xml.1
 /xtables-multi
diff --git a/iptables/Makefile.am b/iptables/Makefile.am
index 8e902ef369b8e..74bc2bf98ef8c 100644
--- a/iptables/Makefile.am
+++ b/iptables/Makefile.am
@@ -63,6 +63,7 @@ man_MANS         = iptables.8 iptables-restore.8 iptables-save.8 \
                    ip6tables-save.8 iptables-extensions.8 \
                    xtables-nft.8 xtables-translate.8 xtables-legacy.8 \
                    iptables-translate.8 ip6tables-translate.8 \
+		   iptables-restore-translate.8 ip6tables-restore-translate.8 \
                    xtables-monitor.8
 CLEANFILES       = iptables.8 xtables-monitor.8 \
 		   iptables-translate.8 ip6tables-translate.8 \
@@ -96,7 +97,7 @@ iptables-extensions.8: iptables-extensions.8.tmpl ../extensions/matches.man ../e
 		-e '/@MATCH@/ r ../extensions/matches.man' \
 		-e '/@TARGET@/ r ../extensions/targets.man' $< >$@;
 
-iptables-translate.8 ip6tables-translate.8:
+iptables-translate.8 ip6tables-translate.8 iptables-restore-translate.8 ip6tables-restore-translate.8:
 	${AM_VERBOSE_GEN} echo '.so man8/xtables-translate.8' >$@
 
 pkgconfig_DATA = xtables.pc
-- 
2.22.0

